### ISSUE TITLE
Add continuous integration

### ISSUE THREAD

**Issue by ndmitchell** [tomjaguarpaw/haskell-opaleye#56]
Opaleye should be tested on multiple GHC versions on every commit.

**Comment by tomjaguarpaw** [comment]
I have added Travis support: https://travis-ci.org/tomjaguarpaw/haskell-opaleye/builds

I still need to add a badge to the README.

**Comment by ndmitchell** [comment]
Cool! I'd be keen on migrating the changes from your copy of `neil` to the master one, if we can. It seems like some of the things you changed you might fix in opaleye then change back.

**Comment by tomjaguarpaw** [comment]
Besides the one I just opened an issue for on `neil` the issues I had were:

* I couldn't get "No correct source-repository link", and "No bug-reports link" to work, no matter what I tried.  I have no idea what went wrong.
* I had false positives for "Missing CHANGES.txt in extra-source-files" and "Missing README.md in extra-source-files" because I have those files in `extra-doc-files` (and mine is called `CHANGELOG.md`).
